### ISSUE TITLE
train_test_split stratification seems to be in reverse

### ISSUE THREAD

**Issue by MINGYUK** [mims-harvard/MedTok#3]
https://github.com/mims-harvard/MedTok/blob/4a8d1016038e8316828163b8c26b63e618eee2b0/MedTok_EHR_Tutorial/MedTok_EHR.py#L166

It looks like stratification should be done on multi-label classification tasks (phenotyping / drug rec) but it is set to none on these tasks, and stratified on labels for other tasks.
